configure.ac

```diff
@@ -29,9 +29,9 @@ AC_INIT([Flatpak],
         [flatpak],
         [http://flatpak.org/])
 
-GLIB_REQS=2.44
+GLIB_REQS=2.54.0
```

Comment:
What's the necessity for this? Generally flatpak tries to be compatible with old GLib versions so it can work across many distros

Reply:
Maybe you could add a copy of that function to `common/flatpak-utils.c` to avoid bumping the GLib requirement, if it's pretty self-contained.